app/uk/gov/hmrc/cataloguefrontend/auth/AuthController.scala

```diff
@@ -70,5 +71,5 @@ object AuthController {
   }
 
   def continueUrl(targetUrl: Call): Call =
-    routes.AuthController.postSignIn(sanitize(Some(RedirectUrl(targetUrl.url))))
+    routes.AuthController.postSignIn(sanitize(Some(RedirectUrl(if (targetUrl.url == "/") "/index.html" else targetUrl.url))))
```

Comment:
Why can't we just use `appRoutes.CatalogueController.index.url` (i.e. `/`)? If we need this, can we move `index.html` into app.routes rather than a hardcoding here?

Note, `postSignIn` takes `Option` so could we use `None` instead? e.g. `Some(targetUrl.url).filterNot(_ == appRoutes.CatalogueController.index.url).map(RedirectUrl.apply)` - but will need a comment saying why we have to do this

Reply:
Thanks fixed.